# Redemption

Removing the broken code passes this test.

Of course, I'm back to an implementation that always "randomly" returns 
the same result.

But I've got an additional passing test that checks for a kind of error 
that I've already demonstrated I might make in the future.



